client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
```

Comment:
Check already exists in https://github.com/Automattic/wp-calypso/blob/master/client/devdocs/index.js#L13.